client/src/less/bhima-bootstrap.less

```diff
@@ -8,6 +8,44 @@
 
 @border-radius-base: 2px;
 
+@ima-green:           #77a13f;
+@ima-dark-orange:     #e57b14;
+@ima-dark-blue:       #085484;
+@ima-light-orange:    #fdba64;
+@ima-light-blue:      #6fbdee;
+  
+/** @todo move all structure code into less with configurable variables */
+@expanded-offset:  260px;
+@collapsed-offset: 56px;
+@nav-height:       51px;
+
+/**
+* Notifications
+*/
+growl-notifications { 
+  width : 100%;
+  position : fixed;
+  top : 0px;
+  left : @collapsed-offset;
+  z-index : 1000;
+}
+
+.expanded > growl-notifications { 
+  left : @expanded-offset;
+  width : calc(~'100% - (@{expanded-offset} - @{collapsed-offset})');
+}
+
+growl-notifications growl-notification {
+  background : #fff;
+  border-left : 5px solid #9A9A9A;
+  height : @nav-height;
+  padding : 15px 10px;
+  display : block;
+
+  /** @todo width calculation are a hack - the notification doesn't need to be fixed */
+  width : calc(~'100% - @{collapsed-offset}');
```

Comment:
Aha, thanks for the info.  I wasn't sure if this was less or CSS, which made it difficult to research.

@IMA-WorldHealth/local-contributors, if you'd like to see the documentation for this, [here is a link](http://lesscss.org/functions/#string-functions-e)